src/featbin/wav-reverberate-test.cc

```diff
@@ -0,0 +1,143 @@
+// featbin/extract-segments.cc
```

Comment:
Do not reimplement convolution and block-convolution. Include them using a header file.

Change the comment at the top of the file, it says 
>// featbin/extract-segments.cc